amundsen_application/static/js/components/NavBar/styles.scss

```diff
@@ -23,6 +23,10 @@
   color: $brand-color-4;
```

Comment:
Changing the existing style to the following should work as well:
```
.nav-bar a:hover,
.nav-bar a:hover span {
  color: $brand-color-4;
}
```

This also has the added benefit of consolidating the styles as well as applying styles based on the element as opposed to the className that it just happens to have right now for the current look-and-feel.

Reply:
ah - yeah that'd definitely be cleaner